lib/segmentio/index.js

```diff
@@ -162,6 +162,7 @@ Segment.prototype.normalize = function(msg){
   msg.userId = msg.userId || user.id();
   msg.anonymousId = user.anonymousId();
   msg.messageId = uuid();
+  msg.sentAt = new Date();
```

Comment:
Ah I guess analytics.js works a bit differently. On mobile we set sentAt right before uploading, so in case it fails, we just retry uploading later and set sentAt again.